kafka-splice-adapter-streaming/REAME.md

```diff
@@ -0,0 +1,31 @@
+Here are steps to make a simple streaming app that will produce rows from the kafka producer and using spark streaming it will insert into the splice engine with the help of splice-spark adapter:
+
+1. Download kafka_2.10-0.10.0.1 and untar it in the home dir.
+
+1. Start kafka server in one window (or start with nohup command and keep it running in background)
+$ bin/kafka-server-start.sh  config/server.properties
+
+2. Register the kafka topic  (Need to do it only once even if kafka server is restarted). e.g. Sample app coonects kafka topic "test-k", soto start "test-k" topic, following command was used:
+$ bin/kafka-topics.sh --zookeeper localhost:2181 --create --topic test-k --partitions 2 --replication-factor 1
+
+4. Keep kafka producer stream ready (kafka-producer/run_prod.sh), it can be used to stream some mock data to splice-adapater app that will use kafka consumer on the same kafka-topic. 
+
+5. If running on kerberized cluster, copy the hbase users keytab to some common dir such as /tmp on all nodes so that app can reference that. Currently, it will require to use hbase/<masterNode> principal and its keytab. It can cab be found from master node  /var/run/cloudera-scm-agent/process/, searching for latest hbase-MASTER dir, hbase.keytab should be present in that dir. (This step will require sudo privelege)
+
+6. Main.java uner src folder contains a sample streaming app that takes the data from kafka stream to the spark stream and then insert the data to splice engine using splice-spark adapter. You will need to get the build artifacts jars and keep it at the common location say /tmp/dependency on all nodes. Also copy spark jars to /tmp/dependency from the installed base (for cloudera from /opt/cloudera/parcels/SPARK2/lib/spark2/jars, make sure spark-streaming jar is also copied)
```

Comment:
What jars are you keeping in /tmp/dependency? I think this should work without that step